### PR TITLE
Update the Gitlab-runner go version to 1.13

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
@kacf made some minor changes. Could you have a second quick look?

File: Makefile [hunk 1]
```diff
@@ -186,8 +186,8 @@ godeadcode:
 	deadcode -ignore version.go:Version
 
 govarcheck:
-	echo "-- checking with varcheck"
-	varcheck .
+	echo "-- Varcheck is deprecated..."
```
Comment (kacf) [inline Makefile]: Has it just moved, or is it really deprecated?

Comment (oleorhagen) [reply Makefile]: lol, moved. Sry. Fixing.

**Comment by oleorhagen** [comment]
Only an echo changed since the last runs:
* https://gitlab.com/Northern.tech/Mender/mender-qa/-/jobs/395517036
* https://gitlab.com/Northern.tech/Mender/mender/pipelines/107478178

**Pull request merged**